LICENSE-WEBRTC

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2011, The WebRTC project authors. All rights reserved.
```

Comment:
You can link to the original repo LICENSE file in the README or indicate that such files contain the corresponding license header at the top. We are using other third libraries and not copying here the license files of them. There is no need to have this file physically here.